tutorials/fundamentals/installation.md

```diff
@@ -1,55 +1,101 @@
-# napari installation tutorial
+---
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.0
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# How to install napari on your machine
+
+Welcome to the **napari** installation guide!
+
+This guide will teach you how to do a clean install of **napari** and launch the viewer.
+
+```{note} 
+If you want to contribute back to the napari codebase, you should follow [this guide instead](https://napari.org/developers/contributing.html).
```

Comment:
```suggestion
If you want to contribute code back into napari, you should follow the [development installation instructions in the contributing guide](https://napari.org/developers/contributing.html) instead.
```

I'd kind of like the words "development installation" included in this sentence, otherwise it's a bit hard to see how this page content relates to this note.